src/Request.js

```diff
@@ -92,13 +102,15 @@ function xmlHttpPost (url, params, callback, context) {
       httpRequest.timeout = context.options.timeout;
     }
   }
+
+  addHeaders(httpRequest, headers);
   httpRequest.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded; charset=UTF-8');
   httpRequest.send(serialize(params));
 
   return httpRequest;
 }
 
-function xmlHttpGet (url, params, callback, context) {
+function xmlHttpGet (url, params, callback, context, headers) {
```

Comment:
thanks for your contribution @delatal!

i assume you tacked on `headers` as an additional method argument to ensure backwards compatibility, but my first instinct is that its placement in the signature is awkward (given that it is _preceded_ by an optional context binding).

what header/s do you need to pass along in your own requests exactly?

Reply:
I just need to add an Authorization header.